test/api/liskApi.js

```diff
@@ -99,25 +100,113 @@ describe('Lisk API module', () => {
 			(LSK).should.be.type('object').and.be.instanceof(LiskAPI);
 		});
 
-		it('should set currentPeer string by default', () => {
-			(LSK).should.have.property('currentPeer').and.be.type('string');
+		it('should set node string by default', () => {
+			(LSK).should.have.property('node').and.be.type('string');
 		});
 
-		describe('with testnet equal to true', () => {
+		describe('with option testnet true', () => {
 			beforeEach(() => {
 				LSK = new LiskAPI({ testnet: true });
 			});
 
-			it('should set the port to 7000', () => {
+			it('should set the port to 7000 on initialization', () => {
 				(LSK).should.have.property('port').be.equal(testPort);
 			});
 
-			it('should set testnet to true', () => {
+			it('should set testnet to true on initialization', () => {
 				(LSK).should.have.property('testnet').be.equal(true);
 			});
 		});
 	});
 
+	describe('on initialize', () => {
+		describe('SSL', () => {
+			it('should set SSL to true on initialization when passed as an option', () => {
+				LSK = new LiskAPI({ ssl: true });
+				(LSK).should.have.property('ssl').be.true();
+			});
+
+			it('should set SSL to false on initialization when passed as an option', () => {
+				LSK = new LiskAPI({ ssl: false });
+				(LSK).should.have.property('ssl').be.false();
+			});
+
+			it('should set SSL to false on initialization when no options is passed', () => {
+				LSK = new LiskAPI({ });
+				(LSK).should.have.property('ssl').be.false();
+			});
+		});
+
+		describe('randomNode', () => {
+			it('should set randomNode to true when no node is given and randomNode not explicitly set', () => {
+				(LSK).should.have.property('randomNode').be.true();
```

Comment:
We should consistently test with `new LiskAPI()` or `new LiskAPI({})` (or both).

Reply:
`new LiskAPI({})` is part of the beforeEach hook (now with curly braces)